src/System.Private.CoreLib/src/System/Diagnostics/StackTrace.CoreRT.Wasm.cs

```diff
@@ -53,7 +53,7 @@ private unsafe void InitializeForCurrentThread(int skipFrames, bool needFileInfo
         internal string ToString(TraceFormat traceFormat)
         {
             if (traceFormat == TraceFormat.Normal && _builder.ToString().EndsWith(Environment.NewLine))
```

Comment:
I would be nice to call `ToString` just once.

Reply:
Right, no problem.  If its ok I'll wait until I get the emscripten 2.0.0 #8281 in, as its a bit of pain resetting emscripten versions as its at 400+MB download each time.